frontend_lib/src/component/Tags/NewTagForm.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import { postContentTag } from '../../action.async.js'
+import { sendGlobalFlashMessage, handleFetchResult } from '../../helper.js'
+
+// require('./NewTagForm.styl') // see https://github.com/tracim/tracim/issues/1156
+
+export class NewTagForm extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      tagName: ''
+    }
+  }
+
+  handleClickBtnValidate = async () => {
+    const { props } = this
+    const response = await handleFetchResult(
+      await postContentTag(props.apiUrl, props.workspaceId, props.contentId, this.state.tagName)
+    )
+    if (!response.ok) {
+      sendGlobalFlashMessage(props.i18n.t('Error while adding a tag to the content'))
+    }
+
+    // CR - 08/06/2021 - TODO - autre fenêtre d'ajout si get 200 ok afficher que le tag est bien added
```

Comment:
@charline-rageade can you translate your comment to english or delete it

Reply:
if it stays here, it needs to be attached to an issue